components/JobCardLoader.vue

```diff
@@ -0,0 +1,31 @@
+<script>
+import { ContentLoader } from 'vue-content-loader'
+
+export default {
+  components: {
+    ContentLoader,
+  },
+}
+</script>
+<template>
+  <content-loader
+    :height="160"
+    :width="400"
```

Comment:
Isn't it breaking the layout? This width shouldn't be fixed.

Reply:
when I did the tests for yarn run did'nt break ... i need to see with you later about being fixed this values

Reply:
The loaders should accept % values.
Please, fix and reopen the PR. Thanks!